frontend/apps/app/features/organizations/pages/OrganizationMembersPage/InviteMemberModal.tsx

```diff
@@ -0,0 +1,164 @@
+'use client'
+
+import { inviteMember } from '@/features/organizations/actions/inviteMember'
+import {
+  Button,
+  Input,
+  ModalActions,
+  ModalContent,
+  ModalDescription,
+  ModalOverlay,
+  ModalPortal,
+  ModalRoot,
+  ModalTitle,
+  useToast,
+} from '@liam-hq/ui'
+import { useRef, useState } from 'react'
+import type { ChangeEvent, FC, FormEvent } from 'react'
+import styles from './InviteMemberModal.module.css'
+
+interface InviteMemberModalProps {
+  isOpen: boolean
+  onClose: () => void
+  organizationId: string
+}
+
+export const InviteMemberModal: FC<InviteMemberModalProps> = ({
+  isOpen,
+  onClose,
+  organizationId,
+}) => {
+  const [email, setEmail] = useState('')
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState<string | null>(null)
+  const [success, setSuccess] = useState(false)
+  const toast = useToast()
+
+  const handleEmailChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setEmail(e.target.value)
+    if (error) setError(null)
+    if (success) setSuccess(false)
+  }
+
+  const validateEmail = (email: string): boolean => {
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/
+    return emailRegex.test(email)
+  }
```

Comment:
Regarding `validateEmail()`, since it's only used for client-side validation, it might be sufficient to rely on the `type="email"` attribute of the `<input>` element. The custom function might be unnecessary, but I'm not entirely sure. It is ok for now!